R/eval-tidy.R

```diff
@@ -19,7 +23,7 @@
 #'   values and throw errors if you try to access non-existent values.
 #'
 #'
-#' @param expr An expression to evaluate.
+#' @param expr An expression or quosure to evaluate.
 #' @param data A data frame, or named list or vector. Alternatively, a
 #'   data mask created with [as_data_mask()] or [new_data_mask()].
```

Comment:
Can you say something briefly about how the contents are positioned wrt the `env`? (i.e. they'll be found earlier)

Reply:
I have added a section about data masking, with a little example.